post_installation_configuration/cluster-tasks.adoc

```diff
@@ -706,3 +706,5 @@ include::modules/installation-images-samples-disconnected-mirroring-assist.adoc[
 include::modules/installation-restricted-network-samples.adoc[leveloffset=+2]
 
 include::modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc[leveloffset=+2]
+
+include::modules/images-cluster-sample-imagestream-import.adoc[leveloffset=+1]
```

Comment:
Should this be `[leveloffset=+2]` to match the Cluster Sample Operator modules before it?